lib/matplotlib/tests/test_colorbar.py

```diff
@@ -1049,3 +1049,42 @@ def test_title_text_loc():
     # colorbar axes, including its extend triangles....
     assert (cb.ax.title.get_window_extent(fig.canvas.get_renderer()).ymax >
             cb.ax.spines['outline'].get_window_extent().ymax)
+
+
+@check_figures_equal(extensions=["png"])
+def test_passing_location(fig_ref, fig_test):
```

Comment:
I don't think this should have the unrelated test as part of the same PR?

Reply:
I can open a separate PR for those. Just thought that I'd "piggyback" them in here after merging a PR without tests...

Reply:
Just don't want the tests, which seem a bit problematic, to slow down this PR ;-)

Reply:
I think I have sorted out the tests. At least in my opinion (wasn't obvious to me, not using images, how the interpolation worked, but turning it off lead to what I expected).

I'll add a note now (and leave the tests in for now).